handler_error.go

```diff
@@ -94,7 +94,11 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 		}
 
 		if e.Spec.Proxy.StripListenPath {
-			r.URL.Path = strings.Replace(r.URL.Path, e.Spec.Proxy.ListenPath, "", 1)
```

Comment:
Are you sure that RawPath should be changed here?
If so does it mean that you solved some bug?